### PR TITLE
Update the Backward Compatibility Service Level Agreement (SLA)

### PR DESCRIPTION
**Pull request by iseeyuan**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #35

We are reaching to a consensus to an SLA window (6 months) for both mobile and server, both internal and OSS. Update it accordingly in this RFC.

### CODE REVIEW

File: RFC-0017-PyTorch-Operator-Versioning.md [hunk 1]
```diff
@@ -19,12 +19,11 @@ This document proposes a new BC and FC policy based on operator versioning.
...
 Backwards compatibility (BC), the ability for PyTorch to continue running programs from older versions, is important so programs don’t need to be forcefully updated to comply with the new runtime implementation.
 
-PyTorch current SLA on backwards compatibility:
+PyTorch SLA on backwards compatibility:
 
 
 
-* **OSS** — “stable” features will be deprecated for one release before a BC-breaking change is made. [PyTorch OSS BC-breaking policy](https://pytorch.org/docs/master/) 
-* **FB Internal** — we will not break a serialized torchscript program running in production at Facebook (to be replaced with a more generic SLA)
+* PyTorch SLA will ensure that models developed using a cerntain version will be supported for *six months* from the version release date.
```
Comment (cccclai) [inline RFC-0017-PyTorch-Operator-Versioning.md:26]: Is 6 months better or 180 days?

**Review by raziel — CHANGES_REQUESTED** [review]
Shouldn't we have a sub-section at the end of the proposal section that describes the new SLA? Rather than modifying the "History" section

**Comment by iseeyuan** [comment]
> Shouldn't we have a sub-section at the end of the proposal section that describes the new SLA? Rather than modifying the "History" section

Good catch! Let me move to the proposal section and update the wording.

**Review by mruberry — APPROVED** [review]
LGTM!

**Comment by raziel** [comment]
Thanks @iseeyuan 

Should we also call out as a goal/outcome that moving forward we're not having a difference between internal and OSS guarantees since they would be moving under the same SLA?

Also, update the Facebook / FB references :-)

**Review by raziel — APPROVED** [review]
Looks good, aside from the point I raised on internal vs oss

**Comment by iseeyuan** [comment]
> Thanks @iseeyuan
> 
> Should we also call out as a goal/outcome that moving forward we're not having a difference between internal and OSS guarantees since they would be moving under the same SLA?
> 
> Also, update the Facebook / FB references :-)

Sounds good! Will call out the outcome, and update "Facebook". Thanks!

**Pull request merged**